pkgs/by-name/pa/payme/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "payme";
+  version = "v1.2.0";
```

```suggestion
  version = "1.2.0";
```

`version` should start with a number.